wcfsetup/install/files/lib/system/user/notification/UserNotificationHandler.class.php

```diff
@@ -1066,4 +1076,23 @@ public function getLatestNotification($lastRequestTimestamp)
 
         return [];
     }
+
+    public function filterUsersByContentLanguage(array $userIDs, int $contentLanguageID): array
```

Comment:
This should either be private or have a PHPDoc comment with an explanation. In any case it should have:

```
/**
 * @param int[] $userIDs
 * @return int[]
 */
```

To properly type the array parameters.